File: 250814/학점 계산기/credit-calculator.py
import sys
input = sys.stdin.readline

n = int(input())
arr = list(map(float, input().split()))
total = 0.0;

for i in range(n):
    total += arr[i]

avg = round(total / n, 1)

print(avg)

if avg >= 4.0:
    print("Perfect")
elif avg >= 3.0:
    print("Good")
elif(avg < 3.0):
    print("Poor")

#print("Perfect" if avg >= 4.0 else "Good" if avg >= 3.0 else "Poor")
#avg round(total/n, 1) 1번째 까지 반올림

# print(f"{avg:.1f}")  # 소수 1자리까지 반올림하여 출력

File: 250814/학점 계산기/credit-calculator.cpp
#include <iostream>
#include <cmath>   //ceil, round, floor
#include <iomanip> //fixed, setprecision
using namespace std;

int main() {
    int N;
    cin >> N;

    double sum { 0.0 };

    double arr[6];
    for(int i{0};i < N;i++){
        cin >> arr[i];
        sum += arr[i];
    }
    double avg = sum / (double) N;
    cout << round(avg * 10) /  10.0 << "\n";

    // 평균 소수 첫째 자리까지 출력 (반올림)
    //cout << fixed << setprecision(1) << avg << "\n"; 


    //소수점 아래를 자름, floor
    int res = static_cast<int>(avg);
    
    string eval;
    switch(res) {
        case 4: eval = "Perfect";
        break;

        case 3: eval = "Good";
        break;
        
        default: eval = "Poor";
    }
    cout << eval << "\n";
    return 0;
}

// double d = 3.14159;
// int i = static_cast<int>(d); // 소수점 이하 버림
